spec/shared/room_spec.rb

```diff
@@ -0,0 +1,383 @@
+shared_examples 'room_behavior' do |room_type, query:|
```

Comment:
I can fix this for Ruby 2.0, if that's preferred. Just adding a value here should suffice.